tests/manual/test_memory_reporting.py

```
@@ -0,0 +1,293 @@
+    vectors_config = info.get("config", {}).get("params", {}).get("vectors", {})
+    sparse_vectors_config = info.get("config", {}).get("params", {}).get("sparse_vectors", {})
+
+    indexed_fields = [
+        name for name, schema in payload_schema.items()
+        if schema.get("points", 0) > 0
+    ]
+
+    dense_vector_names = list(vectors_config.keys()) if isinstance(vectors_config, dict) else []
+    sparse_vector_names = list(sparse_vectors_config.keys()) if isinstance(sparse_vectors_config, dict) else []
+
+    print(f"\nCollection has:")
```

Ruff (0.15.9)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)